docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,104 @@
+This guide was developed using a `Nokia G-240W-B` router. It should work for other models in the `G` series.
+
+### Device information
+
+- Device name: `Nokia G-240W-B`
+- Vendor: `Nokia`
+- Hardware version: `3FE56756ADBA`
+- Software version: `3FE47969AGCA21`
+- Chipset: `BCM6838`
+
+> Note:
+There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
+- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well.
+- The `ISP` provides `IPv6` address, prefix and gateway to the router.
+
+For this example, we will be using the `192.168.1.10` and `fe80::192:168:1:10` addresses for the following steps. You will need to substitute your own addresses in the following steps.
+
+Notice that the `IPv4` and `IPv6` addresses suffixes matches the number `xxx.168.1.10` and `xxxx::192:168:1:10` respectively, this is intentional to recognize the addresses were manually assigned in the `Raspberry Pi`.
+
+Worth to mention that the `IPv6` address that starts with `fe80` is the `link-local` address and it is used in this guide, the other addresses that start with `2806` are the global addresses and they are assigned by the router.
+
+### Grab your IPv4 and IPv6 address from your Raspberry Pi
+
+SSH into your `Raspberry Pi`. A quick `ifconfig eth0` (substituting `eth0` for the applicable network interface) will get you the address information:
+
+```bash
+eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
+        inet 192.168.1.10  netmask 255.255.255.0  broadcast 192.168.1.255
+        inet6 2806:103e:1f:2ace:22a1:de1d:e1bf:30dc  prefixlen 64  scopeid 0x0<global>
+        inet6 2806:103e:1f:2ace:7d21:7337:7817:2fb2  prefixlen 128  scopeid 0x0<global>
+        inet6 fe80::e6f3:5490:577e:4fed  prefixlen 64  scopeid 0x20<link>
+        inet6 fe80::192:168:1:10  prefixlen 64  scopeid 0x20<link>
+        ether e4:5f:01:cd:8e:e1  txqueuelen 1000  (Ethernet)
+        RX packets 4342  bytes 704995 (688.4 KiB)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 3376  bytes 661707 (646.1 KiB)
+        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
+```
+
+e.g. `inet 192.168.1.10  netmask 255.255.255.0` and `inet6 fe80::192:168:1:10 prefixlen 64`
+
+### IPv4 configuration
+
+Go to **Network** --> **LAN**
+
+1. Set `IPv4 address` to the default gateway address, `192.168.1.1` or `192.168.1.254` dependinding on your specific router configuration.
```

Comment:
```suggestion
1. Set `IPv4 address` to the default gateway address, `192.168.1.1` or `192.168.1.254` depending on your specific router configuration.
```

Reply:
That's a horrible typo!
The word is now correct: `depending`
The pics has been updated.